packages/ai/README.md

```diff
@@ -37,10 +37,18 @@ Creates an instance of `EdgeDBAI` with the specified client and options.
 
   Returns a new `EdgeDBAI` instance with an updated query context.
 
-- `async queryRag(message: string, context: QueryContext = this.context): Promise<string>`
+- `async queryRag(message: string, context?: QueryContext): Promise<string>`
 
   Sends a query with context to the configured AI model and returns the response as a string.
 
+- `streamRag(message: string, context?: QueryContext): AsyncIterable<StreamingMessage> & PromiseLike<Response>`
+
+  Used both when you want to handle streaming data as it arrives or when you need a `Response` object that streams the results.
```

Comment:
Might be worth documenting these separately, almost like two overloads?

Reply:
Hey I updated Readme and merged this, if you have time u can check the updated Readme.